AWSTranscribe.podspec

```diff
@@ -8,11 +8,10 @@ Pod::Spec.new do |s|
   s.homepage     = 'http://aws.amazon.com/mobile/sdk'
   s.license      = 'Apache License, Version 2.0'
   s.author       = { 'Amazon Web Services' => 'amazonwebservices' }
-  s.platform     = :ios, '12.0'
+  s.platform     = :ios, '9.0'
   s.source       = { :git => 'https://github.com/aws-amplify/aws-sdk-ios.git',
                      :tag => s.version}
   s.requires_arc = true
   s.dependency 'AWSCore', '2.36.1'
   s.source_files = 'AWSTranscribe/*.{h,m}'
-  s.resource_bundle = { 'AWSTranscribe' => ['AWSTranscribe/PrivacyInfo.xcprivacy']}
 end
```

Comment:
do not merge this PR in until next model update picks up the changes that keep these lines